src/main/java/org/semver4j/Range.java

```diff
@@ -26,6 +26,10 @@ public Semver getRangeVersion() {
         return rangeVersion;
     }
 
+    public boolean isSatisfiedByAny() {
+        return rangeVersion.isEqualTo("0.0.0") && rangeOperator == RangeOperator.GTE;
```

Comment:
Maybe we should extract this "magic" string `0.0.0` to some const. We use it in https://github.com/semver4j/semver4j/blob/87d98cabd6fde09efe01138622468b11e8c52566/src/main/java/org/semver4j/internal/range/processor/GreaterThanOrEqualZeroProcessor.java#L20

I propose in `Semver` class:

https://github.com/semver4j/semver4j/blob/87d98cabd6fde09efe01138622468b11e8c52566/src/main/java/org/semver4j/Semver.java#L17-L19

add const:

```
public static final Semver ZERO = new Semver("0.0.0");
```

This allows us to use it in `isSatisfiedByAny` method:

```
public boolean isSatisfiedByAny() {
    return rangeVersion.isEqualTo(ZERO) && rangeOperator == RangeOperator.GTE;
}
```

And in the `GreaterThanOrEqualZeroProcessor` like this:

```
return format(Locale.ROOT, "%s%s", GTE.asString(), ZERO);
```